### ISSUE TITLE
Focus management is needed for list and loops in the Review screen

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/vets-design-system-documentation#5026]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

I was reviewing a form my team was building and encountered incorrect focus management for list and loops on the review screen

[Link to 8416 on stage](https://staging.va.gov/supporting-forms-for-claims/submit-medical-expense-report-form-21p-8416/) where I first saw this

## What I expected to happen

On the review screen, focus management for the list and loop should match the focus management inside the form flow:

**Edit**
When editing a list and loop item, after the user is done, focus needs to be brought to the success message on the review screen. This is happening very briefly but then focus is lost for some reason. This doesn't happen in the standard edit flow, just when editing from the review page.

**Cancel**
When editing a list and loop item and then deciding to cancel, after the user is done, focus should be brought back to the h4 "Review" heading. Currently focus goes to the H2 stepper

**Delete**
When Deleting a list and loop item, focus should be to be brought to the alert so users know their action was completed successfully. Currently focus is going to the "Add another" button

## Reproducing

- CSS Library version:
- Device: (e.g. iPhone 8, Macbook)
- Browser: (e.g. Firefox, IE 11)

**URL(s) where this bug occurs**

-
-
-

**Steps to reproduce:**

1.
2.
3.
4.

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

**Comment by jeana-adhoc** [comment]
I've filed two bugs for our engineers to work on this ticket
- #5316 
- #5318
